_posts/2024-06-03-so-you-want-better-opensearch-documentation.markdown

```diff
@@ -0,0 +1,60 @@
+
+OpenSearch actually does a pretty good job of the _Reference_ portion of the puzzle, especially considering that there are new releases every six weeks. I started contributing documentation late last year and recently joined the [_opensearch-project/documentation-website_](https://github.com/opensearch-project/documentation-website) repository as a Maintainer. I see a constant flow of both new docs and the improvement of existing docs. Considering the ratio of writers to developers and where the project was not too long ago, I think we’re doing a really good job of keeping up with both. 
+
+The other three quadrants, however, is where we could do a better job. There is _Tutorial_  and _Explanation_ content mixed in with the _Reference_ content, but you can spend a lot of time wading through the minutiae of how things work when you are really looking for understanding of the core principles. We also have a huge amount of great content in the existing [blog](https://opensearch.org/blog/) that could flesh out these other areas of documentation, not to mention if we start linking out to third-party blogs. This is an area I plan to invest more energy into.
+
+Charlie then asked the audience what their thoughts were and we captured a lot of great insights. I then followed up with Heather Halter whose remarks are in *italics*. 
+
+**“Older versions of OpenSearch docs often show up in Google which is often jarring. Can we make sure Google is properly indexing the latest content?“**
+
+_While we’ve made a lot of progress in this area, particularly with implementing a redirect strategy, we are currently planning on adding canonical links to all the older pages. The goal is to get this work done by the end of Q2, which should show a big improvement in the back half of the year._
+
+**“I often find a GitHub issue that answers my question and I ⭐it.” Someone suggested that we look for comments on GitHub issues that have lots of stars and use that as input to where docs could be improved.**
+
+ _Great suggestion!_
+
+**“My 2-yr old won’t let me work at night” was one response to the question of why people who find a gap in documentation don’t pursue opening a PR to improve the docs.** 
+
+_We love community contributions and have made it really easy to start. Check out the [CONTRIBUTING.md](http://contributing.md/) file and watch this [YouTube video](https://www.youtube.com/watch?v=zmVC3fQwnVI) on our documentation process and how you can contribute. You can also search through the issues labeled "[good first issue"](https://github.com/opensearch-project/documentation-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or "[help wanted"](https://github.com/opensearch-project/documentation-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)._
+
+**Backwards compatibility in API’s came up as an issue. It would be interesting if we could better communicate API changes over time.** 
```

Comment:
```suggestion
**Backwards compatibility in APIs came up as an issue. It would be interesting if we could better communicate API changes over time.** 
```